packages/web/src/common/store/profile/sagas.js

```diff
@@ -376,18 +373,6 @@ function* fetchProfileAsync(action) {
       )
     )
 
-    if (!isNativeMobile) {
-      const showArtistRecommendationsPercent =
-        getRemoteVar(DoubleKeys.SHOW_ARTIST_RECOMMENDATIONS_PERCENT) || 0
```

Comment:
should we be respecting this config value somewhere new?

Reply:
iirc it controls the frequency that the follow suggestions popup shows on follow clicked

Reply:
imo i think we should drop this complexity, what do you think?

Reply:
yeah I think that's totally fine, just wanted to be a conscious decision